### DISCUSSION TITLE
No Doorbell Message in Homekit

### DISCUSSION THREAD

**Discussion by eteanton** [tsightler/ring-mqtt#839] [Support Q&A]
Hi everyone,

I've recently switched from Homebridge to Home Assistant thanks to my new NAS. Thanks to various tutorials, I've already learned a lot. However, I'm having trouble properly integrating my Ring Intercom into HomeKit.

My Ring Intercom is connected via Ring MQTT, and I have the lock set up as an accessory in HomeKit. Unlocking works fine, but I'm not receiving notifications in HomeKit when someone rings the doorbell. The Doorbell entity from Ring MQTT detects the ringing, but nothing comes through to HomeKit. What could be the issue?

It worked perfectly in Homebridge (and no, I don't want to switch back 😉).

Thanks for your help

**Comment by tsightler** [comment]
In general you are better off asking Home Assistant and Homekit questions somewhere else.  This forum is about ring-mqtt, and it sounds like ring-mqtt itself is working properly.  Most people that would respond on this thread are not Home Assistant experts and a significantly smaller portion know the details of how Home Assistant exposes device to Homekit.  In general, I recommend using Homebridge for this use case but you've already stated you don't want to switch, although technically, switching isn't required, you can use both ring-mqtt+Home Assistant and Homebridge together just fine.

Regardless, I can try to make some guesses based on my limited understanding of Homekit in general, and Home Assistants method of exposing devices to Homekit.  Based on your description, I believe the issue is that you are attempting to use Ring Intercom and expose it as a doorbell to Homekit.  As far as I understand, the only way to expose a doorbell accessory from Home Assistant to Homekit is to expose a camera and associate it with a binary doorbell sensors using the linked_doorbell_sensor configuration option.  Basically, the Homekit Bridge code appears to assume that a doorbell is always a video doorbell, while Ring Intercom has no video component.

You could potentially create a dummy generic camera that just uses a JPEG image and expose that, associating the Ring Intercom binary sensor with this generic camera device.  That's the best idea I have.

Note that Homekit itself also has a little bit of weirdness around a doorbell accessory that doesn't have a video component.  While it's technically possible to create a doorbell accessory this way (that's what homebridge-ring does), some features, such as disabling chimes on specific Homepods, will not work correctly if there is no video component to the doorbell, even though that limitation makes no real sense.

Good luck!

**Reply by eteanton** [reply]
Thank You for your quick response
